OutSystems/outsystems-ui
# ROU-3075

This PR is for reducing the reflow on the tooltip transition, due to the transitions on the position attribute

![image](https://user-images.githubusercontent.com/32780808/154100503-ffa4f58a-159d-4a5f-ad6d-53d41181ba08.png)


### What was happening

- reflow due to transition of the position attribute on the .osui-tooltip__balloon-wrapper__balloon.

### What was done

- Replaced position with transformX/Y.
- Final position is always trasnform(0)
- Created two new css variables to easily customize the transition length (default is 12px), and avoid lost magic number on the code.
--	--tooltip-transition-distance: 12px;
--	--tooltip-inverted-transition-distance: -12px;

### Test Steps

1. Click all tooltips
2. Check if animation is correct
3. There should be no visible differences to previous implementaiton


### Checklist
